src/accapi/client.py

```diff
@@ -160,48 +151,47 @@ def _receive_realtime_car_udpate(self):
         else:
             self._request_entry_list()
 
-    def _receive_entry_list(self):
-        entryList = EntryList.receive(self._receive)
+    def _receive_entry_list(self, msg):
+        entryList = EntryList.from_message(msg)
         self._cars = {i: self._cars[i] if i in self._cars else -1 for i in entryList.carIndices}
 
-    def _receive_entry_list_car(self):
-        args = EntryListCar.receive_args(self._receive)
+    def _receive_entry_list_car(self, msg):
+        args = EntryListCar.parse(msg)
         car = EntryListCar(*args)
         self._cars[car.carIndex] = len(car.drivers)
         for callback in self._onEntryListCarUpdate.callbacks:
             car = EntryListCar(*args)
             callback(Event(self, car = car))
 
-    def _receive_track_data(self):
-        args = TrackData.receive_args(self._receive)
+    def _receive_track_data(self, msg):
+        args = TrackData.parse(msg)
         for callback in self._onTrackDataUpdate.callbacks:
             data = TrackData(*args)
             callback(Event(self, data = data))
 
-    def _receive_broadcasting_event(self):
-        args = BroadcastingEvent.receive_args(self._receive)
+    def _receive_broadcasting_event(self, msg):
+        args = BroadcastingEvent.parse(msg)
         for callback in self._onBroadcastingEvent.callbacks:
             event = BroadcastingEvent(*args)
             callback(Event(self, event = event))
 
-    def _request_connection(
-        self,
-        displayName: str,
-        password: str,
-        updateIntervalMs: int,
-        commandPassword: str
-    ):
+    def _request_connection(self):
```

Comment:
In #4, _request_connection now uses self attributes as much as possible, only leaving out passwords so they never get stored.